docs/content/components/footer.mdx

```diff
@@ -101,6 +101,12 @@ best_practices: |
   - Only use approved social media icons from the design system
     - Social media icons may vary for English and Spanish
 
+  ### External Links
+  For Footer links that are outside of .gov and other supported sites, an external link icon will be added at the end of the link's title to indicate that the link will go to an external site. This icon is added automatically based on the where link is going to, but can also be overridden to display or be hidden regardless of the link.
+
+  - To override an INTERNAL link to be displayed as EXTERNAL, add the class `external-link`.
```

Comment:
`usa-link--external` already exists, let's make sure we're implementing the same techniques